docs/conf.py

```diff
@@ -182,22 +191,33 @@
     "contributing.rst": "community/index.rst",
 }
 
-# ABlog configuration
-blog_path = "examples/blog/index"
-blog_authors = {
-    "pydata": ("PyData", "https://pydata.org"),
-    "jupyter": ("Jupyter", "https://jupyter.org"),
-}
-
-
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
 html_css_files = ["custom.css"]
 todo_include_todos = True
 
+# -- favicon options ---------------------------------------------------------
+
```

Comment:
Could you add a link to the sphinx favicon config docs as a comment here for reference